sycl/test-e2e/InvokeSimd/invoke_simd_smoke.cpp

```diff
@@ -94,7 +80,7 @@ template <bool use_func_directly> bool test() {
   constexpr unsigned Size = 1024;
   constexpr unsigned GroupSize = 4 * VL;
 
-  queue q(ESIMD_selector_v, createExceptionHandler());
+  queue q(default_selector_v, createExceptionHandler());
```

Comment:
will this still pick the GPU even if multiple devices are installed (cpu.gpu and fpga maybe)? thx

Reply:
It will only select a GPU in that scenario, because of this line https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/InvokeSimd/lit.local.cfg#L4. I Confirmed this on a machine with cpu, and fpga emulator devices.

I can change it to `gpu_selector_v` if that is preferable.

Reply:
nah that seems fine, thanks for checking